mm2src/coins/lp_coins.rs

```diff
@@ -2968,6 +2970,39 @@ pub async fn withdraw(ctx: MmArc, req: WithdrawRequest) -> WithdrawResult {
     coin.withdraw(req).compat().await
 }
 
+pub async fn ibc_withdraw(ctx: MmArc, req: IBCWithdrawRequest) -> WithdrawResult {
```

Comment:
>  I think if we can use the unified withdraw..

We already talked about this with Artem. We decided to have this functionality as seperated rpc commands because of complexity and even security reasons.

> If you think this is not applicable for any reason, please move new cosmos/IBC RPCs to a new folder inside rpc_command similar to lightning RPCs.

This is done